### ISSUE TITLE
Need Update!!

### ISSUE THREAD

**Issue by DevQiao** [patriciogonzalezvivo/thebookofshaders#221]
Thx for your awesome work, when will you update the captures? can't wait :yum:

**Comment by miccall** [comment]
Need Update ，too  .  great  job

**Comment by FirepadCN** [comment]
Thx ,need update,催更🙂

**Comment by NumerHero** [comment]
Nice work, Thx

**Comment by pgmrK** [comment]
Really appreciate this guide to GLSL.
Looking forward to updates as well. 😊

**Comment by einarpersson** [comment]
Is this project still alive? Awesome resource, too bad that the later pars are missing!

**Comment by DevQiao** [comment]
2 years passed after I commented this repository....
Still hoping u to update this😭

**Comment by dromer** [comment]
It seems that most of the updates have gone into translations, typography and rectifications.
Which is of course also awesome!

But indeed a continuation on the, still open, chapters would be great!

**Comment by jcyuan** [comment]
actually I have learnt all the rest parts of the menu from other place in the past years.
a few years passed and it still stops right that... I think there are a lot of people donated for this project to help it move forward include me, but... it's a bit kind of disappointed. anyway thanks for leading me from a good begining into the shader world.

**Comment by patriciogonzalezvivo** [comment]
Sorry for the feeling of disappointment. The donations mostly cover the server cost ~45 a month. I still have plans to start writing chapters again, but life keeps getting in the way (mostly 2 kids and a fulltime job). Is in my dreams to open a GitHub sponsorship so I can quit my job and dedicate full-time to generate content like this book and keep developing tools like GlslViewer, GlslCanvas, GlslEditor, GlslTexture, MidiGyver, Hilma, Hypatia, etc

**Comment by jcyuan** [comment]
> Sorry for the feeling of disappointment. The donations mostly cover the server cost ~45 a month. I still have plans to start writing chapters again, but life keeps getting in the way (mostly 2 kids and a fulltime job). Is in my dreams to open a GitHub sponsorship so I can quit my job and dedicate full-time to generate content like this book and keep developing tools like GlslViewer, GlslCanvas, GlslEditor, GlslTexture, MidiGyver, Hilma, Hypatia, etc

suggest that the new content of the rest parts could be chargable as it's indeede valuable!

**Comment by SMUsamaShah** [comment]
Yes, charge for it. What you have given us for now has created enough desire for more. Will gladly pay.

**Comment by c101088** [comment]
THX for this great work. Suggest that provides the Alipay as a another way of donate.

**Comment by Kemeros** [comment]
Ever thought about making a patreon? Easier to get donations if people subscribe.

**Comment by och-uop** [comment]
There seems to be a nice group of supporters here ❤️ 
I have just arrived but this book looks very interested and I too hope it can be completed.
@patriciogonzalezvivo I wish you will find the time to write again.
all the best

**Comment by summer-go** [comment]
if there is a paid course, i will pay. hope that you will have time to complete the rest chapters.
